std/algorithm/mutation.d

```diff
@@ -2960,12 +2667,12 @@ if (isBlitAssignable!T && !is(typeof(lhs.proxySwap(rhs))))
     swap(a, b);
 
     //Note: we can catch an error here, because there is no RAII in this test
-    import std.exception : assertThrown;
-    void* p, pp;
-    p = &p;
-    assertThrown!Error(move(p));
-    assertThrown!Error(move(p, pp));
-    assertThrown!Error(swap(p, pp));
+    //import std.exception : assertThrown;
```

Comment:
Why not delete instead of commenting?

Reply:
I wanted to see whether this is the only issue.
As explained, the core.lifetime functions are technically different and using them is a breaking change.